jobs/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.9 on 2021-11-24 05:31
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Jobs',
```

Comment:
the model should be named Job and not Jobs. Django adds automatically the plural